src/lib.rs

```diff
@@ -9,5 +9,9 @@ extern crate serde_derive;
 mod net;
 mod packet;
 
+pub mod server;
+pub mod connection;
+pub mod amethyst_error;
```

Comment:
I want to rethink how we do error management and if we want to use the `failure` crate but I don't quite know where the community is heading.